filebrowser/templates/filebrowser/include/filelisting.html

```diff
@@ -36,7 +36,7 @@
 
         <!-- FILESELECT FOR RTE/TINYMCE -->
         <!-- we need to use the absolute url here -->
-        {% if query.pop == "2" or query.pop == "4" %}
+        {% if query.pop in "245" %}
```

Comment:
`query.pop in "245"` condition triggers if `query.pop` is an empty string. Due to that bug, "SELECT" buttons appear on filebrowser home page http://example.com/admin/filebrowser/browse/

You should return the previous condition back